### ISSUE TITLE
Sonomablocker

### ISSUE THREAD

**Issue by MehdiYawari** [Theile/venturablocker#5]
Hi
Sorry for writing about sonomablocker as an issue here. 
Is there a chance you're going to do a Sonoma Blocker this year?
Just like last year your blocker has been really helpful for many macadmins for blocking new macos this some testing is done.
Thanks a lot in Advances

**Comment by Theile** [comment]
I don't think so. When Apple released Venture they released is as an update, so it did not require the download of a full app for the upgrade to be done. The blocker cannot prevent this update. 

It's anticipated that Apple continues with the update for Sonoma, so blocker will not work anyway. 

Also it seems that the OS delay functionality is important to configure on clients, and also prepare the back end for the new OS. 

After all it is in every Apple users best interest to get the new OS out as soon as possible. 

It's more secure than it's predecessor anyway, too.

**Comment by MehdiYawari** [comment]
Yes, we are also hoping to move to macOS Sonoma as soon as possible. Although Apple released macOS Major Update through System Software Update as an update and not installer app and we will be also deferring macOS to give time for all applications to support macOS Sonoma, but I assume, the possibility exits that someone get the macOS Sonoma installer not from System Software Update but from anywhere else.
We will also restrict Install macOS Sonoma.app but it is not a 100% solution. That's why, this sonomablocker should prevent the update in such cases.

**Comment by Cpuroast** [comment]
If the machine is enrolled in an MDM, it still offers the full installer for Ventura, rather than the delta update, which allows VenturaBlocker and an eventual SonomaBlocker to prevent installation.

We've had VenturaBlocker deployed everywhere while being enrolled in Jamf with no extra policies and it's prevented unexpected upgrades, that's why a SonomaBlocker would still be very useful.

**Comment by Theile** [comment]
OK, I shall see if I can do this…

**Comment by Theile** [comment]
Do you mind testing this?
[Sonoma blocker](https://github.com/Theile/sonomablocker/releases/latest)

**Comment by Cpuroast** [comment]
I get a 404 for that link, looks like the entire sonomablocker repo isn't accessible.

**Comment by MehdiYawari** [comment]
Yes I get a 404 error too

**Comment by Theile** [comment]
New release 20230921. Please try this.
